Cargo.toml

```
@@ -4,11 +4,18 @@ version = "0.1.0"
 authors = ["FuriosaAI SW Team <pkg@furiosa.ai>"]
 edition = "2021"
 
+[features]
+blocking = [] # Enable blocking APIs
+
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [[bin]]
 name = "list_npu"
 path = "bin/list_npu.rs"
 
+[[bin]]
+name = "list_npu_blocking"
+path = "bin/list_npu_blocking.rs"
+
```

```suggestion
required-features = ["blocking"]

```
지금 blocking feature를 주지 않으면 컴파일이 실패해서, 의존성을 명시해주면 좋겠습니다.

---

네 그렇네요. 감사합니다.